File: Java/Hardware.java
// Hardware merupakan kelas orang tua kedua dengan urutan
// Produk <- Hardware <- Memory

public class Hardware extends Product{

    // deklarasi atribut

    private String brand = "";
    private String model = "";

    // parameter konstruktor
    public Hardware(String brand, String model) {
        this.brand = brand;
        this.model = model;
    }

    // Konstruktor
    public Hardware() {

    }

    // Methods Set dan get seluruh atribut ..

    public void setBrand(String brand) {
        this.brand = brand;
    }
    public String getBrand() {
        return this.brand;
    }

    public void setModel(String model) {
        this.model = model;
    }
    public String getModel() {
        return this.model;
    }

}

File: Phyton/Hardware.py
from Product import Product

# Hardware merupakan kelas orang tua kedua dengan urutan
# Produk <- Hardware <- Memory

class Hardware(Product):
    
    #  konstruktor
    def __init__(self):
        self.brand = "blank"
        self.model = "blank"
        
    # semua metode set dan gett disetiap atribut

    def setBrand(self, brand):
        self.brand = brand
        
    def getBrand(self):
        return self.brand 

   
    def setModel(self, model):
        self.model = model
        
    def getModel(self):
        return self.model 